tests/core/test_utils.py

```diff
@@ -123,7 +123,6 @@ async def test_error_handling_connection_timeout(caplog):
     assert str(e.value) == str(ONE_WORKING_ONE_TIMEOUT_OUTPUT)
 
     logs = caplog.get_records("call")
-    assert len(logs) == 3
```

Comment:
Okay I'm happy to handle this in another issue since it's not a fatal teardown error. Please comment out this line instead of deleting it, and include a

``` python
# TODO: <link to new issue>
```

Reply:
OK I think I sorted it actually - see the changes to the `RE` fixture and the `AsyncStatus` tests. Although we had some cleanup for these, they didn't seem to take into account pending tasks